test/SourceKit/CursorInfo/cursor_callargs.swift

```diff
@@ -6,8 +6,12 @@ class C1 {
 var c1 = C1(passInt: 0, andThis: 0)
 c1.meth(0, passFloat: 0)
 
-// RUN: %sourcekitd-test -req=cursor -pos=6:11 %s -- %s | %FileCheck -check-prefix=CHECK-CLASS %s
+// RUN: %sourcekitd-test -req=cursor -pos=6:10 %s -- %s | %FileCheck -check-prefix=CHECK-CLASS %s
```

Comment:
Just checking: it should work the same on both column 10 and 11, right?

Reply:
Yeah, no difference - I'll change it back to reduce the diff.